airflow/sensors/filesystem.py

```diff
@@ -43,18 +43,19 @@ class FileSensor(BaseSensorOperator):
     template_fields = ('filepath',)
     ui_color = '#91818a'
 
-    def __init__(self, *, filepath, fs_conn_id='fs_default', **kwargs):
+    def __init__(self, *, filepath, fs_conn_id='fs_default', recursive=False, **kwargs):
```

Comment:
Can you add description about this on Line 41 please

Reply:
@kaxil Sorry, should have noticed the docstring. [90ce4ad](https://github.com/apache/airflow/pull/16894/commits/90ce4adeaa7c61f515266522223498e8b5d0b9cb) adds it.